pages/version-policy/index.md

```diff
@@ -276,49 +276,49 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Confluent-Kafka 2.3.x-4.0.0e</td>
+        <td>Confluent-Kafka 2.5.x-4.1.2</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Confluent-Kafka 2.4.x-4.1.1 (Recommended)</td>
+        <td>Confluent-Kafka 2.6.x-5.1.2 (Recommended)</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Confluent-ZooKeeper 2.3.x-4.0.0e</td>
+        <td>Confluent-ZooKeeper 2.3.x-4.1.3e</td>
```

Comment:
@vishnu2kmohan Is this `Confluent ZooKeeper 2.3.0-4.0.0e` or `Confluent ZooKeeper 2.5.0-4.1.3e`?
![Screen Shot 2019-04-29 at 10 22 12 AM](https://user-images.githubusercontent.com/36425832/56914351-bc4fcc00-6a68-11e9-8192-1ee17d2e4076.png)

Reply:
The latter: `Confluent ZooKeeper 2.5.0-4.1.3e`, thanks for catching the`2.5.0` prefix @anithamuthu79

Reply:
@vishnu2kmohan Would you be able to make this change for me to review and sign-off?

Reply:
Pushed the change. Thanks!